src/rust/catpowder/win/runtime.rs

```diff
@@ -228,6 +217,28 @@ fn count_processor_cores() -> Result<usize, Fail> {
     return Ok(core_count);
 }
 
+/// Deduces the RSS settings for the given interface. Returns the number of valid RSS queues for the interface.
+fn deduce_rss_settings(api: &mut XdpApi, ifindex: u32) -> Result<u32, Fail> {
+    const DUMMY_QUEUE_LENGTH: u32 = 1;
+    let sys_proc_count: u32 = count_processor_cores()? as u32;
+
+    for queueid in 1..sys_proc_count {
```

Comment:
Why is this loop from 1 to sys_proc_count-1? It would be good to add a comment explaining why this is done.

Reply:
Added a comment.